module1/homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,23 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result;
+
+    if (command == "add") {
+        result = std::to_string(first + second);
+    } else if (command == "subtract") {
+        result = std::to_string(first - second);
+    } else if (command == "multiply") {
+        result = std::to_string(first * second);
+    } else if (command == "divide") {
+        if (second = 0) {
```

Comment:
```suggestion
        if (second == 0) {
```
Przypisanie masz zamiast porównania. W if do second przypisujesz wartość 0 zamiast porównać czy second jest równe 0 i jest problem :)

Reply:
I have done some changes in the newest commit, please take a look